interfaces/odata.js

```diff
@@ -19,8 +19,8 @@ declare module 'odata' {
   };
 
   declare type OHandler<TModel> = {
```

Comment:
I don't like it, but at this point, we still use it here: https://github.com/Brewskey/brewskey.admin/blob/dev/app/auth/sagas.js#L46-L56